### ISSUE TITLE
 http_proxy.rb -- breaking proxy request by adding/duplicating data #52 

### ISSUE THREAD

**Issue by do11** [igrigorik/em-proxy#53]
Why ticket #52 is closed while bug is not fixed?

**Comment by igrigorik** [comment]
Because it's an example that's not core to the actual library and based on your feedback, you did not care if it was fixed or not. Hence, I closed it. I may get to it in the future.

If you're willing to take a run at it, please reopen #52 and we can iterate there. In the meantime, closing this.

**Comment by do11** [comment]
How I did not care, If I spend time to find a bug, then, not just ignore it like most people would do, but, like good open source citizen, report it to author, then, as if is not just enough, found _example_ solution, and reported it to author again, and now, I'm convincing you to fix it? And based on this feedback you decide I don't care? If some random user like me (spending all this much time) doesn't care, why author don't care to apply one line fix which he is already know?

These (examples) is, after all, educational material about your code, which people will use to learn it, and it's wrong. Bugs in examples hitting hardest, because people are not mastered exemplified code at full yet.

I can not reopen #52 because there is no reopen button for non-owners of repo:

> ![](http://i.imgur.com/5zdqdQG.png)

And what to iterate there? I already explained all my findings in full detail, my obligations as user and reporter fulfilled on 101%. I will not also study git and github for you to fix 1 line of code. What a strange requirements for fixes? This looks like punishing user who suffered bug in your code even more. You would fix it in 10 seconds long ago and we'll forget happily about it.
